radixdlt-java/src/main/java/com/radixdlt/client/atommodel/Fungible.java

```diff
@@ -0,0 +1,16 @@
+package com.radixdlt.client.atommodel;
+
+import org.radix.utils.UInt256;
+
+/**
+ * Temporary interface for representing an amount of interchangeable Particles.
```

Comment:
Why is this temporary?  What conditions are there for it getting removed?

Reply:
Good question. All the other interfaces quark interfaces are marked temporary as well, so I assume @talekhinezh has some plan to change them in the near future.